perception/object_recognition/prediction/map_based_prediction/launch/map_based_prediction.launch.xml

```diff
@@ -4,9 +4,9 @@
   <arg name="prediction_time_horizon" default="10" />
```

Comment:
Please change this to `10.0` since ROS2 is more strict about handing double value if it is declared as double parameter.

```suggestion
  <arg name="prediction_time_horizon" default="10.0" />
```

Reply:
added in the latest commit